packages/cli/test/simulation/fourNodesForks.test.ts

```diff
@@ -0,0 +1,86 @@
+import chai, {expect} from "chai";
+import chaiAsPromised from "chai-as-promised";
+import {routes} from "@lodestar/api/beacon";
+import {logFilesDir, SimulationEnvironment, FAR_FUTURE_EPOCH} from "../utils/simulation/index.js";
+
+chai.use(chaiAsPromised);
+
+describe.skip("fourNodesForks", function () {
+  this.timeout("5m");
+  let env: SimulationEnvironment;
+
+  const testCases: {
+    title: string;
+    params: {
+      event: routes.events.EventType;
```

Comment:
Since this test always wait for finalization, not need to declare the same event everytime